db/migrate/20170216193758_add_vm_table_comments.rb

```diff
@@ -0,0 +1,81 @@
+class AddVmTableComments < ActiveRecord::Migration[5.0]
+  def change
+    change_table_comment :vms, "Virtual Machines and Templates"
+
+    change_column_comment :vms, :autostart, "A boolean value that indicates whether or not the VM is set to autostart."
```

Comment:
hmm, hope I'm not undoing what other people said. But stating the type doesn't seem helpful to me.

Maybe

```ruby
change_column_comment :vms, :autostart, "true if the VM is set to autostart."
```